### PR TITLE
Updates to layout of registries

### PR DESCRIPTION
**Pull request by SteveLLamb**
Updates to menu as per discussion with @japierce1028. 

Intro for Terms to be added.

### CODE REVIEW

**Comment by SteveLLamb** [comment]
closes #76

**Review by palemieux — APPROVED** [review]

**Comment by SteveLLamb** [comment]
@japierce1028, just waiting on your approval to merge this.

**Review by japierce1028 — APPROVED** [review]

**Pull request merged**
